templates/tools/dockerfile/php_deps.include

```diff
@@ -4,4 +4,8 @@
 # Install dependencies
 
 RUN apt-get update && apt-get install -y ${'\\'}
-    git php5 php5-dev phpunit unzip
+    git php5 php5-dev unzip
+
+RUN wget https://phar.phpunit.de/phpunit-5.7.27.phar && \
```

Comment:
If you replace `\` with `${'\\'}`, the linebreaks will work better

Reply:
modified